test/flow-utility-types.js

```diff
@@ -2,11 +2,12 @@
 import assert from 'assert'
 import path from 'path'
 import {
+  degenEnum,
   degenField,
   degenList,
   degenObject,
   degenString,
-} from '../src/generator.js'
+} from '../src/index.js'
```

Comment:
Good catch!

Reply:
I figured as long as I was in there I'd fix that part of it. A pass should probably be done to fix the other couple of cases of `generator.js` being imported, but I've kind of been saving that for getting a full on test framework in place.